### PR TITLE
[Components with more than 1 service] LPS-179751 Refactor ScreenNavigationCategory components from Echo and Security modules

### PR DESCRIPTION
**Pull request by RenanUmV**
[LPS-179751](https://issues.liferay.com/browse/LPS-179751)

### CODE REVIEW

**Comment by RenanUmV** [comment]
ci:test:echo

**Comment by RenanUmV** [comment]
ci:test:security

**Comment by RenanUmV** [comment]
ci:test:sf

**Comment by RenanUmV** [comment]
ci:test:relevant

**Comment by RenanUmV** [comment]
ci:test:echo

**Comment by RenanUmV** [comment]
ci:test:security

**Comment by Kvale1733** [comment]
@RenanUmV No regressions.

File: modules/dxp/apps/multi-factor-authentication/multi-factor-authentication-web/src/main/java/com/liferay/multi/factor/authentication/web/internal/frontend/taglib/servlet/taglib/MFASetupUserAccountScreenNavigationEntry.java [hunk 1]
```diff
@@ -43,6 +41,7 @@
  * @author Marta Medio
  */
 public class MFASetupUserAccountScreenNavigationEntry
+	extends MFASetupUserAccountScreenNavigationCategory
```
Comment (dantewang) [inline modules/dxp/apps/multi-factor-authentication/multi-factor-authentication-web/src/main/java/com/liferay/multi/factor/authentication/web/internal/frontend/taglib/servlet/taglib/MFASetupUserAccountScreenNavigationEntry.java:44]: @RenanUmV This is not a component, no need to extend the `Category`. In this case, the protected `Language` field is never initialized in the `Entry` instance, and it's actually not used either.

File: modules/dxp/apps/multi-factor-authentication/multi-factor-authentication-web/src/main/java/com/liferay/multi/factor/authentication/web/internal/frontend/taglib/servlet/taglib/MFASetupUserAccountScreenNavigationCategory.java [hunk 2]
```diff
@@ -37,24 +30,19 @@
...
+	implements ScreenNavigationCategory {
 
 	@Override
 	public String getCategoryKey() {
 		return MFASetupUserAccountScreenNavigationConstants.CATEGORY_KEY_MFA;
 	}
 
-	@Override
-	public String getEntryKey() {
-		return null;
```
Comment (dantewang) [inline modules/dxp/apps/multi-factor-authentication/multi-factor-authentication-web/src/main/java/com/liferay/multi/factor/authentication/web/internal/frontend/taglib/servlet/taglib/MFASetupUserAccountScreenNavigationCategory.java:52]: @RenanUmV It seems the MFA `Entry` services are registered on demand in `MFASetupUserAccountCheckerTracker`, but I'm not sure if this empty `Entry` implementation affects the UI.

Could you ask QA to check whether the UI still works as expected (no change with this PR)?

**Review by dantewang — CHANGES_REQUESTED** [review]
@RenanUmV Please see my comment and resend, thanks!

**Pull request closed without merging**